test/cmdlineTests/optimizer_BlockDeDuplicator/output

```diff
@@ -80,10 +80,6 @@ sub_0: assembly {
         /* "optimizer_BlockDeDuplicator/input.sol":138:174  function f() public { true ? 1 : 3;} */
     tag_3:
       stop
-        /* "optimizer_BlockDeDuplicator/input.sol":108:133  function fun_() public {} */
-    tag_4:
-      jump(tag_3)
-        /* "optimizer_BlockDeDuplicator/input.sol":138:174  function f() public { true ? 1 : 3;} */
```

Comment:
This bit was just silly :-).

Reply:
Interesting. Isn't that the main job of the block deduplicator?

Reply:
It's no duplicate - it's a "forwarding jump", but it's unique. Without the deduplicator things would probably be much worse and we'd even have several of these...

Reply:
But doesn't the deduplicator handle those jumps? Or just if they are really going forward and not backwards?

Reply:
Ah, ok, It does? If so (or even if only in some cases), then I wasn't aware (I still don't really know the old-optimizer steps in all details so far). In any case apparently these it did miss.

Reply:
Maybe it doesn't...